src/styles/blocks/hover-phone.scss

```diff
@@ -0,0 +1,34 @@
+.header {
+  &__hover-blok {
+    display: flex;
+    align-items: center;
+  }
+
+  &__phone-hover {
+    position: absolute;
+    color: $white;
+    min-width: 200px;
+    right: 140px;
+    text-align: right;
+    line-height: 27px;
+    align-self: flex-start;
+    top: -30px;
+    visibility: hidden;
```

Comment:
when using `visibility: hidden`, u will not get an animation(transition), because it appears immediately. Use `opacity` for this